vscode-dotnet-runtime-library/src/Utils/CommandExecutor.ts

```diff
@@ -438,7 +438,8 @@ with options ${JSON.stringify(options)}.`));
 
             const commandResult : proc.SpawnSyncReturns<string> = proc.spawnSync(command.commandRoot, command.commandParts, options);
 
-            if(os.platform() === 'win32')
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+            if(os.platform() === 'win32' && options?.runtimeExtensionKillTask === true)
```

Comment:
Are you killing the bundle process? That may not necessarily trigger a proper rollback because the individual installers are running under the installer service. Also, some installers may have rollback boundaries. just calling that out as you may end up in a bad state with partially installed pieces.

Reply:
I thought I was but NO. 

I added something to send the SIGTERM response on a timeout of 10 minutes by default for when users didnt interact with the prompt. We use passive and quiet so that doesnt happen much. However, the installer taking more than 10 minutes, being unresponsive, and our code assuming it's failed, does. Then we try to retry and fail because another installer is still in use by the old process. That's why I added that.

Turns out it seems to never have worked as intended. The PID we get here is the CMD PID. The installer is spawned by the CMD and there doesnt seem to be a super easy way to get the PID without spawning some shell to run it and getting that shell's PID which is whole other thing.

Im just going to remove it.